config/game_options.txt

```diff
@@ -257,9 +257,6 @@ EVENTS_MIN_PLAYERS_MUL 1
 ## Allow the AI job to be picked.
 ALLOW_AI
 
-## Secborg ###
```

Comment:
![wat](https://user-images.githubusercontent.com/32391752/45241667-944f0500-b2ed-11e8-92bd-382ca362f060.jpg)

Reply:
just remove the `#` before `DISABLE_SECBORG`

Reply:
configs are serverside anyway, this PR is just for discussion purposes